### ISSUE TITLE
defaultValue with input checkbox

### ISSUE THREAD

**Issue by rileylnapier** [oreqizer/redux-forms#20]
Hi, I am trying to use

```
<Field
            name="unsubscribed"
            defaultValue={props.unsubscribed}>
            <input type="checkbox" />
</Field>
```

where props.unsubscribed is a boolean.  it gets mad at me because proptypes for defaultValue needs to be a string.  I cast the boolean to a string, but it doesn't seem to work well with the checkbox.

Can you help?

**Comment by oreqizer** [comment]
must be a bad prop type, will fix in a next version. thanks!

**Comment by oreqizer** [comment]
should be fixed in 0.11.3, please reopen if issue still persists!
